modules/ossm-defining-namespace-receive-sidecar-injection-cluster-wide-mesh-cli.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-defining-namespace-receive-sidecar-injection-cluster-wide-mesh-cli_{context}"]
+= Defining which namespaces receive sidecar injection in a cluster-wide mesh using the CLI
```

Comment:
```suggestion
= Defining which namespaces receive sidecar injection in a cluster-wide mesh by using the CLI
```
IBM style: "Using: To avoid ambiguity, replace this gerund with either by using or that uses."

Reply:
done